src/shared/actions/transfers.js

```diff
@@ -433,7 +433,10 @@ export const makeTransaction = (seedStore, receiveAddress, value, message, accou
     const withPreTransactionSecurityChecks = () => {
         // Progressbar step => (Validating receive address)
         dispatch(setNextStepAsActive());
-
+        //Check the last trit for validity
+        if(isLastTritZero(address) === false){
```

Comment:
Small stylistic change
```suggestion
        if (isLastTritZero(address) === false){
```